docker/conda/environments/requirements.txt

```diff
@@ -11,6 +11,7 @@ nvidia-pyindex
 # Duplicated in conda dev to ensure parity with libprotobuf
 protobuf==4.21.*
 pytest-benchmark>=4.0
+pytest-kafka==0.6.0
```

Comment:
Since this is a dev dependency, it should go in the pip section of the conda environment. (`pytest-benchmark` should be moved too)